cloudbuild/cleanup.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+sleep 2h
+echo "Execution timeout reached; deleting self"
+NAME=$(curl -X GET http://metadata.google.internal/computeMetadata/v1/instance/name -H 'Metadata-Flavor: Google')
```

Comment:
These should probably be quoted, because Google bash style says to always quote command substitutions

Reply:
I updated the quoting.